### ISSUE TITLE
valueFormatter for series tooltip not working

### ISSUE THREAD

**Issue by chgauert** [MyHomeMyData/ioBroker.flexcharts#57]
valueFormatter cannot be used with flexcharts. It is supposed to be a callback function in form of 

valueFormatter: (value) => '$' + value.toFixed(2) 

As i understand so far functions will not make it in/throught JSON.stringify(...) , so in the resulting string (containing the options for the chart) the valueFormatter entry is missing completly.

Is there any possibility to get to work?

flexcharts 0.2.0

**Comment by MyHomeMyData** [comment]
Yes, your right. JSON.stringify() is removing the function definition and prevents it from working as supposed.

I was not aware about this restriction. Unfortunately, I've no idea how to fix it within the presently used concept of flexcharts. Do you got an idea/proposal?

**Comment by chgauert** [comment]
no idea for now, maybe i see/find something while playing around with my charts.
But at least everything else is working so far. i had several problems with echart adapter - but with flexchart i could finally create the charts i wanted. Great job and thanks for your work

**Comment by MyHomeMyData** [comment]
@chgauert I found a solution to forward functions to echarts. It's available in branch [develop](https://github.com/MyHomeMyData/ioBroker.flexcharts/tree/develop).
![image](https://github.com/user-attachments/assets/3345485e-279a-441f-b6c0-b706d3de8f3f)

There are some small changes necessary on user side. Please refer to new section in [readme](https://github.com/MyHomeMyData/ioBroker.flexcharts/tree/develop?tab=readme-ov-file#using-functions-within-definition-of-chart).

Would be great if you can do some testing.

**Comment by chgauert** [comment]
your solution looks good to me, all i wanted to do is now possible (add unit to tooltip, remove english 1000' sign  from numbers 1,000 W -> 1000 W)
besides the plain formating (value) => value + "W" is also tried a few other function declarations and all worked as expected.
I did not test anything with "chart from state" as i only use scripts to create the charts.
And thanks for the solution - now i can add the units to the tooltips

**Comment by MyHomeMyData** [comment]
Functionality is now available (version 0.3.0) via Beta repository.
